test/test_helper.rb

```diff
@@ -1,7 +1,9 @@
-require 'simplecov'
+require 'coveralls'
```

Comment:
Won't `coveralls` have to be added as a development dependency in the spec as well?